roles/bootstrap/tasks/setup_cron.yaml

```
@@ -32,3 +32,10 @@
     minute: "0"
     hour: "5"
     job: (cd {{ repository_folder }} && {{ envvar_prefix }} npm run tools:cleaner)
+- name: Create cron job to execute generate_mongo_stats.sh daily
+  become_user: "{{ server_user_name }}"
+  ansible.builtin.cron:
+    name: Execute generate_mongo_stats.sh daily
```

J'ai un peu galéré mais quand on lance le playbook on fini avec seulement le cron pour la preprod et plus pour la prod (si on part d'un inventory avec d'abord prod puis preprod). Il semblerait qu'il faille rajouté `{{ item.name }}` dans le name du cron afin que ansible puisse différentier les deux : 
```suggestion
    name: Execute generate_mongo_stats.sh daily {{ item.name }}
```